cityofaustin/atd-vz-data
# Clean up atd_txdot_locations table

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/16779

The only columns we need to keep:
location_id
description
last_update
latitude
longitude
location_group
scale_factor
geometry
street_level
is_intersection (per Xavier's request)
council_district - used in `locations_with_crash_injury_counts` view that powers the Locations page in the VZE

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Checkout this branch, start up your local db and run the migrations.
2. Make sure the `atd_txdot_locations` table only has the columns in the list above
3. Run the VZE locally, make sure nothing broke with the removal of these columns. Test the locations page, click on a location. The aerial map doesnt work locally (for me.. does it work for yall?) so dont expect it to be there.
4. Test updating the crash coordinates of a crash in the VZE.


---
#### Ship list
